examples/http_request_error_interceptor/http_request_error_interceptor.description

```diff
@@ -0,0 +1,5 @@
+// `RequestErrorInterceptor` is used to handle the errors occurred in the interceptor pipeline execution. 
+// When an error returned from a `RequestInterceptor` service, intercetpor pipeline execution jumps to the 
+// nearest `RequestErrorInterceptor`. However, if there is no `RequestErrorInterceptor` in the pipeline then  
+// the error response is retunred to the client without executing the actual resource in the target service.
```

Comment:
We can be a bit specific and say 500 error response.

Reply:
For the data-binding failures in the interceptor resources, we are returning a `400 - Bad Request` error response.

Reply:
I see. In that case we can keep it as it is.